maven-surefire-common/src/main/java/org/apache/maven/plugin/surefire/SurefireProperties.java

```diff
@@ -71,13 +71,22 @@ public SurefireProperties( KeyValueSource source )
         }
     }
 
+    @Override
+    public synchronized void putAll(Map<?, ?> t) {
```

Comment:
We have different conventions. Please see the spaces around brackets and new lines.
The checkstyle plugin would crash the build. Did you try to run "mvn install -P run-its"?
It would take quite long time to complete the build, cca one hour.

Reply:
Please keep using one commir in PR.
I recommend you to do `git commit--amend` and then forced push `git push origin +master`.